test/clj/rems/api/test_api.clj

```diff
@@ -52,14 +52,14 @@
       (doseq [user ["owner" "alice" "malice"]]
         (testing user
           (is (response-is-ok? (api-response :get "/api/catalogue/" nil
-                                             "43" user))))))
-    (testing "> api key with whitelist can only impersonate given users >"
-      (doseq [user ["alice" "malice"]]
-        (testing user
-          (is (response-is-ok? (api-response :get "/api/catalogue/" nil
-                                             "44" user)))))
-      (is (response-is-unauthorized? (api-response :get "/api/catalogue/" nil
-                                                   "44" "owner")))))
+                                             "43" user)))))))
+  (testing "> api key with whitelist can only impersonate given users >"
+    (doseq [user ["alice" "malice"]]
+      (testing user
+        (is (response-is-ok? (api-response :get "/api/my-applications/" nil
+                                           "44" user)))
+        (is (response-is-unauthorized? (api-response :get "/api/my-applications/" nil
+                                                     "44" "owner"))))))
```

Comment:
This block was perhaps accidentally moved one level higher than before?

Reply:
I dont understand?

Reply:
`(testing "> api key with whitelist can only impersonate given users >"` used to be inside `(testing "api key user whitelist"`, now it's beside it. Probably an unintentional change.

Reply:
Yes this is what I mean.

Reply:
If it's unclear I can do the change and you can check the diff.